modules/distr-tracing-config-storage.adoc

```diff
@@ -133,6 +153,12 @@ Minimum deployment = 16Gi*
 |`ZeroRedundancy`(no replica shards), `SingleRedundancy`(one replica shard), `MultipleRedundancy`(each index is spread over half of the Data nodes), `FullRedundancy` (each index is fully replicated on every Data node in the cluster).
 |
 
+|elasticsearch:
+  useCertManagement:
+|Use to specify whether or not {JaegerShortName} should use the certificate management feature of the Red Hat Elasticsearch Operator.  This feature was added to {logging-title} 5.2 in {product-title} 4.7 and is the preferred setting for new Jaeger deployments.
+|`true`/`false`
+|`true`
```

Comment:
@JStickler @pavolloffay I am bit confused here.
I see you have marked cert-management feature as `tech-preview`, but why do you apply tech-preview feature as default option?
and why do you upgrade the existing installation to new cert management [1]

[1] - https://github.com/jaegertracing/jaeger-operator/blob/v1.34.1/pkg/upgrade/v1_31_0.go

![image](https://user-images.githubusercontent.com/1004403/173055624-246028c2-4a69-487d-a90d-65b108a04bf9.png)

Reply:
@jkandasa see if how I explained it in the [Release Notes PR](https://github.com/openshift/openshift-docs/pull/45350) makes sense.

Reply:
@JStickler thanks for the Release Notes link. There is a typo there. I have updated in the merged PR :)